src/priceHistory/saga.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Price,
+  fetchPriceHistoryFailure,
+  fetchPriceHistoryStart,
+  fetchPriceHistorySuccess,
+} from 'src/priceHistory/slice'
+import Logger from 'src/utils/Logger'
+import { ensureError } from 'src/utils/ensureError'
+import { fetchWithTimeout } from 'src/utils/fetchWithTimeout'
+import { safely } from 'src/utils/safely'
+import networkConfig from 'src/web3/networkConfig'
+import { call, put, spawn, takeLatest } from 'typed-redux-saga'
+
+const TAG = 'priceHistory/saga'
+
+/** @beta - Exclude from Knip dep check */
+export async function fetchTokenPriceHistory(
```

Comment:
probly worth adding a unit test for this

Reply:
added in 47f933ccf89a32ede28d88f650e2e80743e876ad!